library/core/src/primitive_docs.rs

```diff
@@ -832,7 +832,9 @@ mod prim_array {}
 #[doc(alias = "[")]
 #[doc(alias = "]")]
 #[doc(alias = "[]")]
-/// A dynamically-sized view into a contiguous sequence, `[T]`. Contiguous here
+/// A dynamically-sized view into a contiguous sequence, `[T]`.
+///
+/// Contiguous here
 /// means that elements are laid out so that every element is the same
```

Comment:
Still less than 100 characters.
```suggestion
/// Contiguous here means that elements are laid out so that every element is the same
```